### ISSUE TITLE
[v3] Can we use default router as default locale?

### ISSUE THREAD

**Issue by arvinxx** [shuding/nextra#2771]
for example. If I have a `/blog` url,I want to use en locale as the default locale. and with other locale I can choose to use the `/zh` folder

**Comment by canisminor1990** [comment]
up

**Comment by dimaMachina** [comment]
i guess only via next/js `rewrites` https://nextjs.org/docs/pages/api-reference/next-config-js/rewrites

**Comment by arvinxx** [comment]
@dimaMachina No, rewrite can't achieve this. you can give it a try. There is a miss in topbar and sidemenu
